### PR TITLE
Set agent domain from agent host url & Update help command of quickstart.sh

### PR DESCRIPTION
**Pull request by w-t-yang**
Set agent domain from agent host url & Update help command of quickstart.sh

### CODE REVIEW

File: scripts/quickstart.sh [hunk 1]
```diff
@@ -93,16 +93,35 @@ conf() {
...
+        sed -i "s|$1=.*|$1=$2|" $env_file
     else
         echo "$1=$2" >> $env_file
     fi
+
+    # Extract SUPERBLOCKS_AGENT_DOMAIN from SUPERBLOCKS_AGENT_HOST_URL
+    if [ "$1" = "SUPERBLOCKS_AGENT_HOST_URL" ]; then
+        if [[ $2 == http* ]]; then
+            domain=$(awk -F/ '{print $3}' <<<"$2")
+            conf "SUPERBLOCKS_AGENT_DOMAIN" $domain
```
Comment (frankgreco) [inline scripts/quickstart.sh:105]: We also have "__SUPERBLOCKS_AGENT_DOMAIN" that must always point to the Superblocks Cloud domain. Perhaps this variable is too similar to that one?

Comment (w-t-yang) [reply scripts/quickstart.sh:105]: I didn't know that one. Do you have any suggestion here?

Comment (w-t-yang) [reply scripts/quickstart.sh]: Renamed "SUPERBLOCKS_AGENT_DOMAIN" => "SUPERBLOCKS_AGENT_CUSTOM_DOMAIN"

**Review by avocader — COMMENTED** [review]
Tested it and got an error:
```
sudo ./scripts/quickstart.sh conf SUPERBLOCKS_AGENT_HOST_URL YOUR.VALID.DOMAIN.COM
sed: 1: "/etc/superblocks.conf": unterminated substitute pattern
```

**Comment by w-t-yang** [comment]
> Tested it and got an error:
> 
> ```
> sudo ./scripts/quickstart.sh conf SUPERBLOCKS_AGENT_HOST_URL YOUR.VALID.DOMAIN.COM
> sed: 1: "/etc/superblocks.conf": unterminated substitute pattern
> ```

Did you test on Mac?
I just tested on EC2 with AmazonLinux, there is no problem. 
```
[ec2-user@ip-172-31-95-99 ~]$ sudo superblocks conf SUPERBLOCKS_AGENT_HOST_URL YOUR.VALID.DOMAIN.COM
[ec2-user@ip-172-31-95-99 ~]$ cat /etc/superblocks.conf 
SUPERBLOCKS_AGENT_HOST_URL=YOUR.VALID.DOMAIN.COM
SUPERBLOCKS_AGENT_CUSTOM_DOMAIN=YOUR.VALID.DOMAIN.COM
[ec2-user@ip-172-31-95-99 ~]$ sudo superblocks conf SUPERBLOCKS_AGENT_HOST_URL YOUR.VALID.DOMAIN.COM
[ec2-user@ip-172-31-95-99 ~]$ cat /etc/superblocks.conf 
SUPERBLOCKS_AGENT_HOST_URL=YOUR.VALID.DOMAIN.COM
SUPERBLOCKS_AGENT_CUSTOM_DOMAIN=YOUR.VALID.DOMAIN.COM
[ec2-user@ip-172-31-95-99 ~]$ sudo superblocks conf SUPERBLOCKS_AGENT_HOST_URL https://YOUR.VALID.DOMAIN.COM
[ec2-user@ip-172-31-95-99 ~]$ cat /etc/superblocks.conf 
SUPERBLOCKS_AGENT_HOST_URL=https://YOUR.VALID.DOMAIN.COM
SUPERBLOCKS_AGENT_CUSTOM_DOMAIN=YOUR.VALID.DOMAIN.COM
[ec2-user@ip-172-31-95-99 ~]$ sudo superblocks conf SUPERBLOCKS_AGENT_HOST_URL YOUR.VALID.DOMAIN.COM
[ec2-user@ip-172-31-95-99 ~]$ cat /etc/superblocks.conf 
SUPERBLOCKS_AGENT_HOST_URL=YOUR.VALID.DOMAIN.COM
SUPERBLOCKS_AGENT_CUSTOM_DOMAIN=YOUR.VALID.DOMAIN.COM
```

The BSD(MacOS) version of `sed` expects an extension after flag "-i"
in the script, `sed -i "s|$1=.*|$1=$2|" $env_file` has to be changed to `sed -i "" "s|$1=.*|$1=$2|" $env_file` on MacOS

The whole script is created for Linux distributions. I'm not sure if it's needed to support MacOS at this time.
other commands, e.g."sudo superblocks start", won't work on MacOS as well, because it will try to install Linux Docker

**Review by avocader — APPROVED** [review]

**Pull request merged**